### ISSUE TITLE
New maintainer

### ISSUE THREAD

**Issue by thepriyanshujangid** [PixelOS-AOSP/official_devices#280]
## Name and Codename of the device you want to apply for 
```
poco x5 pro  (redwood)
```

## Device Tree sources

```
https://github.com/thepriyanshujangid/android_device_xiaomi_redwood_pixelos
https://github.com/thepriyanshujangid/proprietary_vendor_xiaomi_redwood
https://github.com/thepriyanshujangid/android_kernel_xiaomi_redwood
```

## What ROM's do you maintain officially/unofficially
```
EvolutionX (official)
DroidX-UI (official)
pixelOS     (unofficial)
```

## How long have you been building Custom ROMs
```
april 2023
```

## Any Exceptions/special concern?
```
No
```

## List of changes/patches applied to source, if any
```
No
```

### Telegram username
```
thepriyanshujangid
```
### Link to your unofficial PixelOS Build Post other than XDA

```
https://t.me/poco_x5_pro_roms/36402   (Android 13)
https://t.me/poco_x5_pro_roms/42390 (Android 14)

```

**Comment by kawaaii** [comment]
Thank you for your application to join the PixelOS team, while we were looking through the application, we noticed a few things.

1. We noticed that you had originally shipped KernelSU with your kernel and reverted before the application, something we strongly advise against.

2. We noticed that there are no original changes or contributions to the device tree, something that we have outlined as a necessity for maintainership.


At this time, we unforntunately cannot accept your application, but best of luck and feel free to apply again in the future after some time has passed.
